### PR TITLE
Enable shell-rendering

### PR DESCRIPTION
**Pull request by ajacksified**
👓 @schwers 

Depends on https://github.com/reddit/node-platform/pull/1

### CODE REVIEW

File: src/app/actions/commentsPage.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ export const fetchCommentsPage = commentsPageParams => async (dispatch, getState
   const commentsPageId = paramsToCommentsPageId(commentsPageParams);
   const commentsPage = state.commentsPages[commentsPageId];
 
-  if (commentsPage) { return; }
+  const { shell } = state;
+
+  if (commentsPage || shell) { return; }
```
Comment (schwers) [inline src/app/actions/commentsPage.js]: 🔕  I'm a little unsure if the data-fetching actions should take shell into account, versus the route handlers. The benefit would be data-fetching code could then be concerned only with how to fetch data, and less about 'when'. Then handlers could have better control of what happens when it's a shell page, say if we wanted partial shell'ing. But I don't have strong feelings either way

**Comment by schwers** [comment]
Works, code looks good. I do have a question on https://github.com/reddit/node-platform/pull/1 that could impact this, so I'll hold the fish for now

**Comment by ajacksified** [comment]
💇

**Comment by ajacksified** [comment]
💇

**Comment by schwers** [comment]
:fish:

**Pull request merged**